### ISSUE TITLE
[Knora-app resource viewer mockup] Hoovering over a property triggers the comments

### ISSUE THREAD

**Issue by mrivoal** [dasch-swiss/knora-app#266]
**Is your feature request related to a problem? Please describe.**
Great! I was looking forward to it since I have asked our users to fill in every property comments (and they didn't see the point of it because it wasn't displayed anywhere in Salsah). 

**Describe the solution you'd like**
However, I think I would prefer if the comments were displayed on mouseovering of a "?" icon, at the very end of each property name (e.g.: " Title (?)"... ). They would be more obvious this way.

Mouseovering a property could also be used to display metadata about the property, such as the creator of the value, the creation date, etc. In Salsah, metadata could be displayed by overing over a "i" icon after the value of the property but you may not want to add a new icon there. 

**Describe alternatives you've considered**

**Additional context**

![Capture d’écran 2020-02-25 à 10 20 45](https://user-images.githubusercontent.com/17294309/75233069-9003f980-57b8-11ea-9f18-c565ca1ff114.png)

**Comment by mrivoal** [comment]
I misunderstood what the comment of a value is. I see now that it is the property `knora-base:valueHasComment` attached to a `knora-base:Value`.

What I mentioned above when I suggested this information to be display by mouseovering a `?` icon was the `rdfs:comment` of an `owl:ObjectProperty` which provides a definition of the use of a property within the context of a given research project.

```
elites-cio:hasConfession a owl:ObjectProperty ;
    rdfs:label "Confession [17]"@en,
        "Confession [17]"@fr ;
    knora-base:objectClassConstraint knora-base:ListValue ;
    knora-base:subjectClassConstraint elites-cio:PersonMemberOfIOC ;
    salsah-gui:guiAttribute "hlist=<http://rdfh.ch/lists/0114/confession>" ;
    salsah-gui:guiElement salsah-gui:List ;
    rdfs:comment "Belief or belonging to a religion"@en,
        "Croyance ou appartenance à une religion"@fr ;
    rdfs:subPropertyOf knora-base:hasValue .
```

If we ask users to provide such a definition (as it is "good practice" for data documentation and reuse), it should be displayed somewhere in Knora-app.

To sum up I think 3 informations should be displayed in relation with a property/value:

- the `rdfs:comment` under a `?` icon, just after the name of the property as per: (Title `?`)
- the `knora-base:valueHasComment` (I am learning today that this property exists)
- metadata attached to the value: who created it, when.

**Comment by flavens** [comment]
@mrivoal Would it make sense to display in the tooltip the metadata attached to the value and the comment `rdfs:comment`?

**Comment by mrivoal** [comment]
If the use of the property `rdfs:comment` is restricted to the one described in https://github.com/dasch-swiss/knora-api/issues/1606#issuecomment-590814549, I think it could make sense, yes. At least we could test it this way.

**Comment by flavens** [comment]
it is now part of the mockups v4 and the DSP-APP UI
